README.md

```diff
@@ -114,6 +114,49 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+#### Publishing messages with schema
+
+The Ably Kafka Connector supports messages which contain schema information by converting them to JSON before publishing them to Ably.
+For example, if messages on the Kafka topic are serialized using Avro with schemas registered at https://<your-schema-registry-host>, then set the following configuration so that those messages are converted from Avro to JSON:
```

Comment:
While I think this Avro example is useful, we should remember that there's nothing Avro-specific about this connector. We support Kafka Connect schemata, which can be generated using either the Avro, JSON or Protobuf converters. We should probably link to the [Kafka Connect Schema Registry docs](https://docs.confluent.io/platform/current/schema-registry/connect.html) so that users can find the appropriate config for their setup?